templates/publishing/src/blog-app.html

```diff
@@ -40,6 +40,7 @@
 
       :host {
         --app-primary-color: #404040;
+        -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
```

Comment:
From the thread I'm still not clear why do we need `-webkit-tap-highlight-color: transparent;`.  iOS recommends to [disable tap highlighting, set the alpha to 0](https://developer.apple.com/library/mac/documentation/AppleApplications/Reference/SafariWebContent/AdjustingtheTextSize/AdjustingtheTextSize.html).  And it seems to me `-webkit-tap-highlight-color: rgba(0, 0, 0, 0)` is enough and works on our supported iOS Safari browsers. 

Is there any browser we support that we need to use `-webkit-tap-highlight-color: transparent`?